imports/plugins/core/orders/server/no-meteor/resolvers/Refund/index.js

```diff
@@ -0,0 +1,10 @@
+import amount from "./amount";
+import createdAt from "./createdAt";
+
+export default {
+  _id: (node) => node._id,
+  amount,
+  createdAt,
+  paymentDisplayName: (node) => node.paymentDisplayName,
+  reason: (node) => node.reason
```

Comment:
Neither of these last two props should be necessary. If GraphQL doesn't find a resolver for it, it will just use the object property with that name, exactly what you're doing here.

Reply:
Does that mean `_id` also isn't necessary?

Reply:
nevermind, commented before I saw your other comment below.